### PR TITLE
Add basic MutableItemPackage.

### PR DESCRIPTION
**Pull request by hiltonc**

### CODE REVIEW

File: LDSContent/ItemPackage.sql [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    UNIQUE(position)
+    );
+        CREATE INDEX subitem_doc_id_index ON subitem (doc_id);
+
+CREATE VIRTUAL TABLE subitem_content_fts USING fts4(subitem_id, content_html, tokenize=HTMLTokenizer %1$@);
+
+CREATE VIEW subitem_content AS
+    SELECT rowid AS _id, c0subitem_id AS subitem_id, c1content_html AS content_html FROM subitem_content_fts_content;
+
+CREATE TABLE subitem_content_range (
```
Comment (stephanheilner) [inline LDSContent/ItemPackage.sql]: With this basically only being the start and end index of paragraph tags now, maybe its a good opportunity to name this table better? Maybe `subitem_paragraph_range`?

File: LDSContent/ItemPackage.sql [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    );
+        CREATE INDEX related_content_item_subitem_index ON related_content_item (subitem_id);
+
+CREATE TABLE related_audio_item (
+    _id INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+    subitem_id INTEGER NOT NULL,
+    media_url TEXT NOT NULL,
+    file_size INTEGER NOT NULL,
+    duration INTEGER NOT NULL
+    );
```
Comment (stephanheilner) [inline LDSContent/ItemPackage.sql]: Would it make sense to add the column for voice gender now, or should we wait until we have data from C&M?

**Comment by stephanheilner** [comment]
Looks good, 1 question, 1 suggestion.

File: LDSContent/ItemPackage.sql (hunk 1 above)
Comment (hiltonc) [inline LDSContent/ItemPackage.sql]: Good call, I opened https://code.ldschurch.org/jira/browse/GLC-3 to track this.

File: LDSContent/ItemPackage.sql (hunk 2 above)
Comment (hiltonc) [inline LDSContent/ItemPackage.sql]: I think we should wait. I could try to predict what the data will look like now... at best we'd have the structure in place early, at worst I'd have to redo it.

**Comment by stephanheilner** [comment]
:+1:

**Pull request merged**